examples/bs4.R

```diff
@@ -1,6 +1,6 @@
 library(magrittr)
 library(cohortBuilder)
-library(shinyCohortBuilder)
+pkgload::load_all()
```

Comment:
```suggestion
library(shinyCohortBuilder)
```

We should keep `library(shinyCohortBuilder)` in the examples.

Reply:
done